Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -75,6 +75,11 @@
         Blunt: 2.5
   - type: DamageOtherOnHit
     staminaCost: 8
+    # PIRATE START
+  - type: CanHoldAccessories
+    whiteListTags: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)